parsec/core/fs/userfs/userfs.py

```diff
@@ -237,6 +243,9 @@ async def set_user_manifest(self, manifest: LocalUserManifest) -> None:
         await self.storage.set_user_manifest(manifest)
 
     async def _instantiate_workspace_storage(self, workspace_id: EntryID) -> WorkspaceStorage:
+        if self._workspace_storage_nursery is None:
```

Comment:
What's the need for this ?

(besides, I cannot find any more occurrences of `self._workspace_storage_nursery` in the codebase so we should remove this variable entirely)

Reply:
It's like the comment bellow, it is used only in `UserFS.run`.

You suggest to remove it because there is no point of having a nursery with only one coroutine ?